index.html

```diff
@@ -5,47 +5,58 @@
 		<meta name="viewport" content="width=device-width, initial-scale=1.0">
 		<title>Jonas Meidinger</title>
 		<link rel="stylesheet" href="styles.css">
+		<link rel="assets" href="/Assets/logos/">
+		<link rel="profile-pic" href="/Assets/1008.jpg">
 	</head>
 	       
 	<body>
 		<main>
-			<section>
-				 <button>
-				 </button>
-				 <p>
-					"ZITAT"
+			<header class="header">
+				 <!--<div class="button-wrap"><button>Push the Button</button></div>
+				 -->
+				 <article class="quote-wrapper">
+				 <hr/>
+				 <p class="quote">
+					Veränderte Marktverhältnisse führen zu der Erfordernis einer dynamischen Anpassung. 
+					Zum Beispiel ist die Digitale Transformation überall angekommen, speziell auf dem Deutschen Markt 
+					aber noch nicht genügend umgesetzt
 				 </p>
-				<img></img>
-			</section>
+				 <hr/>
+				 <p class="quote-caption">-Jonas Meidinger</p>
+				</article>
+				<div><img class="profile-img" src="/Assets/1008.jpg" alt="profile-picture" />
+				</div>
```

Comment:
```suggestion
		<img class="profile-img" src="/Assets/1008.jpg" alt="profile-picture" />
```
Why are you using a div here? can you also apply the styling you are trying to get to the image element itself?
Otherwise try wrapping it into a figure element :)